.github/workflows/oscal-check.yaml

```
@@ -0,0 +1,57 @@
+# SPDX-FileCopyrightText: Copyright 2025 The Open Source Project Security Baseline Authors
+# SPDX-License-Identifier: Apache-2.0
+---
+name: Validate OSCAL Output
+
+on:
+  pull_request: {}
+  workflow_dispatch: {}
+
+permissions:
+  contents: read
+
+jobs:
+  verify:
+    name: Setup and Generate
+    runs-on: ubuntu-latest
+    steps:
+      - name: clone-baseline
+        id: clone-baseline
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+            with:
```

```suggestion
        with:
```

Whoops. Mis-indented my suggestion.